index.js

```diff
@@ -168,6 +168,9 @@ module.exports = {
     // Disabling this rule until this is resolved https://github.com/yannickcr/eslint-plugin-react/issues/1848
     // at the moment the fix makes the code look messy and at times unreadable
     'react/jsx-one-expression-per-line': 'off',
+
+    // Disabling this AirBnB-enabled rule because it creates unnecessary boilerplate
```

Comment:
maybe
```suggestion
    // tests are more reliable, reduce unnecessary boilerplate explicitly setting props to `undefined`, 
    // prevent false-confidence for those less experienced with how default props work
```
to give more details in the file rather than having to find and read the PR

(adapted from your description, feel free to re-word to your style 😅)

I'm fine to merge this PR if other reviewers don't see this suggestion adding value 🤙